### ISSUE TITLE
Unable to get writable property store for this property

### ISSUE THREAD

**Issue by RaoHammas** [pierresprim/Windows-API-Code-Pack#19]
I'm trying to write `ShellObject` property value. But always getting following error.    

**Unable to get writable property store for this property.**    
**Access is denied. (0x80070005 (E_ACCESSDENIED))**   

I've tried running in Administrator mode but still get this error. This is how i'm using it.    

```
private bool SetAppAsFav(ShellObject app)
{
    app.Properties.System.Keywords.Value = new []{"Fav"};
    return true;
}
```    

Thank you :)

**Comment by pierresprim** [comment]
@RaoHammas Unfortunately, some properties are not supported by all ShellObjects, such as Keywords, even if all system properties are accessible from ShellObject.Properties.System. To know if a property is readable and/or writable for a ShellObject given at runtime, you have to browse the ShellObject.Properties.DefaultPropertyCollection to make sure that this property is supported by that ShellObject. If you want to know that at build time, you have to browse the Shell Property System MS Doc ([https://docs.microsoft.com/en-us/windows/win32/properties/windows-properties-system](https://docs.microsoft.com/en-us/windows/win32/properties/windows-properties-system)) for the property you want to handle. As far as I know, the Keywords property is not intented to be used with program files.

So, as this is a Windows limitation, I'm sorry, I can't do anything...

**Comment by pierresprim** [comment]
@RaoHammas Btw, please note that all the shell properties that handle an array, including Keywords, are currently not working because of the bug fixed in #18. The next update (planned for February) will fix that issue.

**Comment by RaoHammas** [comment]
@pierresprim  alright. Got it. Thank you :)
